test/resources/testPage.html

```diff
@@ -82,6 +82,6 @@
                     </span>
     Condition: <link property="itemCondition" href="http://schema.org/UsedCondition"/>Previously owned,
       in excellent condition
-    <link property="availability" href="http://schema.org/InStock"/>In stock! Order now!</span>
+    <link property="availability" content="http://schema.org/InStock"/>In stock! Order now!</span>
```

Comment:
@andypang : Minor one. Can we have a test case for both cases? "href" and "content".